src/module.c

```diff
@@ -7820,6 +7826,7 @@ void moduleReleaseGIL(void) {
  * so notification callbacks must to be fast, or they would slow Redis down.
  * If you need to take long actions, use threads to offload them.
  *
+ * Warning: keymiss notification is a special type
```

Comment:
not sure that warning should be placed here.
i don't see any details about something "special" in the link provided here.
i guess the warning you added above is enough.

Reply:
Ohh, I wanted to put it here but then decided that its better to put it above and forgot to delete it...